app/views/dashboard/districts/show.html.erb

```diff
@@ -1,3 +1,6 @@
+<div class="alert alert-primary">
+  <span class="fw-bold">Report early access:</span> Thanks for your patience as we improve this page
```

Comment:
I think we can make this alert a bit more clear:

```suggestion
  <span class="fw-bold">Early access:</span> This page is under active development, thanks for your patience as we work on improving it
```

Reply:
Thanks for the feedback @rsanheim !